AnkiDroid/src/main/java/com/ichi2/anki/analytics/UsageAnalytics.java

```diff
@@ -388,6 +388,7 @@ public static boolean isEnabled() {
         public static final String OPENED_DISCORD = "Opened Discord";
         public static final String OPENED_FACEBOOK = "Opened Facebook";
         public static final String OPENED_TWITTER = "Opened Twitter";
+        public static final String EXCEPTION_REPORT = "Exception report";
```

Comment:
Maintains consistent capitalization with other events

```suggestion
        public static final String EXCEPTION_REPORT = "Exception Report";
```